src/hero/Hero.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import "./hero.css";
+import cloud from "./images/cloud.png";
+import moon from "./images/moon.svg";
+import { Link } from "react-router-dom";
+const Hero = () => {
+  return (
+    <div className="hero">
+      <img
+        src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/1231630/moon2.png"
+        alt=""
+        className="moon"
+      />
+      <img
+        className="stars"
+        src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/1231630/stars.png"
+        alt="stars"
+      />
+      <img
+        src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/1231630/twinkling.png"
+        className="twinkling"
+        alt="twinkling"
+      />
+      <img className="clouds" src={cloud} alt="cloud" />
+      {/* <div className="stars"></div>
+      <div className="twinkling"></div>
+      <div className="clouds"></div> */}
+
+      <div className="intro">
+        <div className="hero-heading">
+          <h1>
+            <img src={moon} className="moonIcon" />
+            Moonlight
+          </h1>
+        </div>
+        <div className="intro-info">
+          <p>
+            MoonShoot is a component library that gives you the building blocks
```

Comment:
1. There is a discrepancy in the naming of library
2. The heading, URL of the library and various other things suggest that the name of library is Moonlight. However, here it is Moonshot.
3. Forgive me, if this seems as nitpicking